Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -35,16 +35,16 @@ const iconAndText = css`
   overflow: hidden;
   text-align: left;
   cursor: pointer;
-  width: 100%;
 
   label: ProjectTreeItemContainer;
 `;
 
-const content = css`
+const projectTreeItem = css`
   outline: none;
   display: flex;
   align-items: center;
   height: 24px;
+  padding-left: 4px;
```

Comment:
I would determine the indent using dynamic css where it takes a depth parameter of how deep the item is in the tree and then calculates and indent.  However, this tree uses the summary/detail HTML elements which I think it is nesting, so the css approach might not work as the tree isn't a singular flattened list.

Reply:
Yeah, the summary/detail elements make figuring out correct indentation a pain. This padding, though, is to give the icons in each item a little space to the left, otherwise they run up against the edge of the dark blue selected-item-background area and it looks a little awkward.